src/SFML/Audio/SoundFileWriterWav.cpp

```diff
@@ -191,8 +188,9 @@ void SoundFileWriterWav::close()
         m_file.flush();
 
         // Update the main chunk size and data sub-chunk size
-        Uint32 dataChunkSize = static_cast<Uint32>(m_sampleCount * m_channelCount * 2);
-        Uint32 mainChunkSize = dataChunkSize + 36;
+        Uint32 fileSize = m_file.tellp();
```

Comment:
No compiler warning about type conversion here?

Reply:
Our VS CI builds have warnings for it.

```src\SFML\Audio\SoundFileWriterWav.cpp(191) : warning C4244: 'initializing' : conversion from 'std::streamoff' to 'sf::Uint32', possible loss of data```